ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -402,3 +402,12 @@
               (dissoc s key)))
           service
           service-level-keys))
+
+(defn is-service-owner?
+  "Admin can see services that they don't own. So we need to know, if user is a service owner"
+  [app]
+  (let [service-operator-id (get-in app [:transport-service ::t-service/transport-operator-id])
+        is-owner? (some (fn [item]
+                          (if (= service-operator-id (get-in item [:transport-operator ::t-operator/id])) true false))
+                        (get app :transport-operators-with-services))]
+    (if is-owner? true false)))
```

Comment:
returning is-owner? would be enough.

Reply:
some returns first instance that matches with the check.
So the is-owner? could be null. Which isn't nice.
I'll rename that variable.

Reply:
Yes, that is true. is-owner? would be misleading because of the possible nil value. You could use a predicate any? or not-any? if you want to make sure that the return value is a boolean.